### PR TITLE
Siddhi/dice game

### LINKED ISSUE #72: Dice game !
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
I would like to add dice game in which there are 2 players one player will roll the dice and then other and the one who gets maximum number wins.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by siddhi-244**
Fixes #72

### CODE REVIEW

**Review by kanishkaa24 — CHANGES_REQUESTED** [review]
The game is not working. Maybe there is some error with the source of html file.

File: index.html [hunk 1]
```diff
@@ -218,6 +218,17 @@ <h3 id="pop">Have a break? Play a game!</h3>
             </a>
             <p class="mt-3"><strong>PacMan Game</strong></p>
         </div>
+          <div class="col-12 col-md-3">
+            <a href="dice.html" target="_blank">
```
Comment (kanishkaa24) [inline index.html:222]: Your index file is named as dicee.html. Please correct this. Also try to make it more interactive.

**Review by kanishkaa24 — CHANGES_REQUESTED** [review]
The nav bar is distorted in the game window.

**Comment by siddhi-244** [comment]
> The nav bar is distorted in the game window.

Can you tell me exactly where is the problem?

**Comment by kanishkaa24** [comment]
image

**Comment by siddhi-244** [comment]
@kanishkaa24  now please check

**Review by kanishkaa24 — CHANGES_REQUESTED** [review]
image
It is still the same and your game is not working.

**Comment by siddhi-244** [comment]
@kanishkaa24  now?

**Review by geekymeeky — CHANGES_REQUESTED** [review]
Kindly fix this overlapping layout issue in your game.
image

**Comment by siddhi-244** [comment]
> Kindly fix this overlapping layout issue in your game.
> image

I have changed the source in my js file as well as html file.

**Comment by siddhi-244** [comment]
Done

**Review by geekymeeky — CHANGES_REQUESTED** [review]
Player 1 wins even when Player 2 dice has a greater number.
image

**Comment by siddhi-244** [comment]
Done

**Comment by geekymeeky** [comment]
> Player 1 wins even when Player 2 dice has a greater number.
> image

@siddhi-244 The above issue has not been resolved yet. Kindly do it
image

**Comment by siddhi-244** [comment]
i saw the logic,it is right. i dont know what problem is occurring. will you please help me?

**Comment by kanishkaa24** [comment]
Try this:
- Create a start button. DO NOT DISPLAY THE DICE UNLESS THE START BUTTON IS CLICKED.
- Initialise both randomnumber1 and randomnumber2 from zero.
- When the start button is clicked, the on click button will call the random number generator function.
- Then you can have the comparison.
- There might be some issue with image sources so you can also use conditional statements rather than using the source (you can print the random number and check the images).
- Do not initialise the dice with 6.

**Comment by siddhi-244** [comment]
> Try this:
> 
> * Create a start button. DO NOT DISPLAY THE DICE UNLESS THE START BUTTON IS CLICKED.
> * Initialise both randomnumber1 and randomnumber2 from zero.
> * When the start button is clicked, the on click button will call the random number generator function.
> * Then you can have the comparison.
> * There might be some issue with image sources so you can also use conditional statements rather than using the source (you can print the random number and check the images).
> * Do not initialise the dice with 6.

Okayy !

**Pull request closed without merging**